### ISSUE TITLE
Not specifying exactly 2 values for the `indexes` field during a swap returns a non helpful message.

### ISSUE THREAD

**Issue by gmourier** [meilisearch/meilisearch#2972]
**Describe the bug**

Not specifying exactly 2 values for the `indexes` field during a swap returns a non-helpful message.

Can we have a better `message` value with the `bad_request` `code`? 

**To Reproduce**

## Request

`POST`- `swap-indexes`

```json
[
    {
        "indexes": ["movies", "movies_new", "test"]
    }
]
```

## Response

```json
{
    "message": "The json payload provided is malformed. `trailing characters at line 3 column 39`.",
    "code": "malformed_payload",
    "type": "invalid_request",
    "link": "https://docs.meilisearch.com/errors#malformed_payload"
}
```

**Expected behavior**

The DX would be better with a message indicating exactly what is going on and helping the user to correct his request. 

**Meilisearch version:** `v0.30.0-new-index-scheduler.beta.1`

**Comment by curquiza** [comment]
Thanks @gmourier, we will see if we can improve this during pre-release

**Comment by curquiza** [comment]
Fixed by #2763
